service-netty/src/main/java/info/bitrich/xchangestream/service/netty/NettyStreamingService.java

```diff
@@ -295,6 +295,10 @@ protected void handleChannelMessage(String channel, T message) {
     }
 
     protected void handleChannelError(String channel, Throwable t) {
+        if (!channel.contains(channel)) {
```

Comment:
Shouldn't this be channels.containsKey(channel) ?

Reply:
I think so, yes. Fixed under #191, which is awaiting review.